src/readers.jl

```diff
@@ -269,6 +269,13 @@ function parse_doc(document::String, format::NotebookInput)
     srctext = "\n" * join(cell["source"], "")
 
     if cell["cell_type"] == "code"
+      if haskey(cell["metadata"], "jupyter")
+        if cell["metadata"]["jupyter"]["source_hidden"]
+          opt_string = " echo=false"
+        end
+      else
+        opt_string = ""
+      end
```

Comment:
```suggestion
      opt_strings = String[]
      haskey(cell["metadata"], "jupyter") && get(cell["metadata"]["jupyter"], "source_hidden", false) && (push!(opt_strings, "echo = false"))
      opt_string = join(opt_strings, ", ")
```
would be better in terms of future improvement and safer dictionary accessing